CHANGELOG.md

```diff
@@ -222,10 +292,143 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+- EmsRefresh task name to use demodularize classname [(#16594)](https://github.com/ManageIQ/manageiq/pull/16594)
+- ContainerServicePortConfigs using :container_service :name [(#16454)](https://github.com/ManageIQ/manageiq/pull/16454)
+- Establish a new connection instead of reconnect! [(#18010)](https://github.com/ManageIQ/manageiq/pull/18010)
+- Added access to MyTasks to self_service roles [(#18006)](https://github.com/ManageIQ/manageiq/pull/18006)
+- Delegate verify_ssl & verify_ssl= to default endpoint [(#18001)](https://github.com/ManageIQ/manageiq/pull/18001)
+- Fixed error with replication setup when default exclude list used [(#17999)](https://github.com/ManageIQ/manageiq/pull/17999)
+- Fix to handle the case when default value of internal is nil [(#17995)](https://github.com/ManageIQ/manageiq/pull/17995)
+- Only reload settings for servers in current region [(#17992)](https://github.com/ManageIQ/manageiq/pull/17992)
+- Prevent queueing things for a zone that doesn't exist in the region [(#17987)](https://github.com/ManageIQ/manageiq/pull/17987)
+- Add regex for dialog password fields. [(#17986)](https://github.com/ManageIQ/manageiq/pull/17986)
+- Move `virtual` definition for `default_security_group` to OpenStack [(#17979)](https://github.com/ManageIQ/manageiq/pull/17979)
+- Remove line from metering_container_image_spec and FIX CI failure [(#17974)](https://github.com/ManageIQ/manageiq/pull/17974)
+- Load all the values along with their key names into the field update method [(#17973)](https://github.com/ManageIQ/manageiq/pull/17973)
+- Fix Physical Disks save inventory [(#17972)](https://github.com/ManageIQ/manageiq/pull/17972)
+- Fix Canisters save inventory [(#17966)](https://github.com/ManageIQ/manageiq/pull/17966)
+- Add Switch model to RBAC [(#17964)](https://github.com/ManageIQ/manageiq/pull/17964)
+- Consolidate production env checks [(#17957)](https://github.com/ManageIQ/manageiq/pull/17957)
+- Add replication set-up methods to be queued on UI side [(#17956)](https://github.com/ManageIQ/manageiq/pull/17956)
+- Adds task to subservices for correct retirement completion [(#17912)](https://github.com/ManageIQ/manageiq/pull/17912)
+- Enable cancel operation for service template transformation plan request [(#17825)](https://github.com/ManageIQ/manageiq/pull/17825)
+- Add Ownership and Tenancy Mixins to Authentication [(#17731)](https://github.com/ManageIQ/manageiq/pull/17731)
 
 ### Removed
+- Remove todo with virtual attr inclusion in attr list [(#18019)](https://github.com/ManageIQ/manageiq/pull/18019)
+- Remove Debug Message [(#17796)](https://github.com/ManageIQ/manageiq/pull/17796)
+- remove 'require_nested :ConfigurationWorkflow' from parent space [(#17782)](https://github.com/ManageIQ/manageiq/pull/17782)
 - Remove ui_lookup_for_title() [(#17691)](https://github.com/ManageIQ/manageiq/pull/17691)
 
+## Gaprindashvili-5 - Released 2018-09-07
```

Comment:
Why is this also adding Gaprindashvili-5? I'm not sure I understand what's supposed to be in which changelog by branch ...